src/pages/criteria-page/CriteriaForm.tsx

```diff
@@ -39,34 +37,54 @@ const ThoughtLeadershipContentPeerReview = importMDX.sync('./help-texts/peer-rev
 const PresenceContentPeerReview = importMDX.sync('./help-texts/peer-review/PresenceContent.mdx');
 const DescriptionContentPeerReview = importMDX.sync('./help-texts/peer-review/DescriptionContent.mdx');
 
+const fragment = graphql`
+  fragment CriteriaForm_review on PersonReviewNode {
+    sahabinessComment
+    sahabinessRating
+    problemSolvingComment
+    problemSolvingRating
+    executionComment
+    executionRating
+    thoughtLeadershipComment
+    thoughtLeadershipRating
+    leadershipComment
+    leadershipRating
+    presenceComment
+    presenceRating
+  }
+`;
+
 interface OwnProps {
-  initialValue?: CriteriaFormData;
+  review: CriteriaForm_review$key | null;
   isSelfReview: boolean;
-  onSubmit: (data: CriteriaFormData) => void;
-  user: CriteriaForm_user$key | null;
+  onSubmit: (data: CriteriaFormValue) => void;
 }
 
 type Props = FCProps<OwnProps>;
 
 export function CriteriaForm(props: Props) {
   const { onSubmit, isSelfReview } = props;
   const components = useContext(MDXContext);
-  const user = useFragment(fragmentUserNode, props.user);
+  const review = useFragment(fragment, props.review);
   const reviewType = isSelfReview ? 'self' : 'peer';
 
+  // if review properties are null, replace it with undefined
+  let value: CriteriaFormValue = {};
+  for (const key in review) {
+    Object.assign(value, { [key]: review?.[key as keyof CriteriaFormValue] ?? undefined });
```

Comment:
simply assign value to each key
```suggestion
    value[key] = review?.[key as keyof CriteriaFormValue] ?? undefined
```

Reply:
That would throw a Typescript error